linera-service/src/cli_wrappers/docker_wrapper.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use pathdiff::diff_paths;
+use tokio::process::Command;
+
+use crate::util::CommandExt;
+
+use super::util::get_github_root;
+
+pub struct DockerWrapper;
+
+impl DockerWrapper {
+    pub async fn build(bin_path: &str, image_name: &str) -> Result<String, anyhow::Error> {
```

Comment:
I personally don't think this would make sense because when we do `Docker build` we're building a docker image, not a docker container. And `DockerInstance` seems confusing, because as I mentioned in a different comment, you can have either a docker image, or a docker container (that can be running or not).
So I think these methods are just docker interactions, and are pretty separated from each other, which is why I've kept them static. Lmk what you think

Reply:
@andresilva91 Can you propose a set of struct definitions that makes sense, then, along the same lines?

Reply:
Talked during meeting, I understand what you meant now. Will address.